tests/test_endpoints.py

```diff
@@ -408,9 +403,9 @@ def test_receive_app(self):
         Endpoint._startup()
         tep = Endpoint.from_name('test-endpoint')
 
-        self.assertEqual(tep.relations[0].received_app_raw, {'simple': 'value', 'complex': '[1, 2]'})
+        self.assertEqual(tep.relations[0].received_app_raw, {'app-key': 'value'})
         self.assertEqual(tep.relations[1].received_app_raw, {})
-        self.assertEqual(tep.relations[0].received_app, {'simple': 'value', 'complex': [1, 2]})
+        self.assertEqual(tep.relations[0].received_app, {'app-key': 'value'})
```

Comment:
The above removes part of the test data set - specifically the more complex component?